client/src/components/TestSuite/TestSuiteDetails/TestGroupCard.tsx

```diff
@@ -27,11 +37,13 @@ const TestGroupCard: FC<TestGroupCardProps> = ({
         </span>
         <span className={styles.testGroupCardHeaderText}>{runnable.title}</span>
         <TestRunButton
+          buttonText={buttonText}
           runnable={runnable}
           runTests={runTests}
           testRunInProgress={testRunInProgress}
         />
       </div>
+      <Box margin="20px">{description}</Box>
```

Comment:
Needs to be conditional on description existing, else the margin is always there